apps/auth/main.py

```
@@ -74,8 +88,13 @@ def register(request: Request, data: models.RegisterData):
     users.insert_one(user_data)
     return {"status": "success", "message": "User registered successfully"}
 
+if DOCKER == "True":
+    host = "127.0.0.1" # Internal only because it's going to be in a Docker network
+else:
+    host = "0.0.0.0"
```

Ruff (0.8.2)

Use ternary operator `host = "127.0.0.1" if DOCKER == "True" else "0.0.0.0"` instead of `if`-`else`-block

(SIM108)